README.md

```diff
@@ -35,17 +33,11 @@ configuration.
 
       Save your CA's intermediate certificates as *cert/cabundle.pem*
 
-5. When asked to provide a Common Name for the server enter the domain name the server will be running on. I.e.
-`yourdomain.com`. If running a development version of the server on OSX you can also use your Bonjour name i.e.
-`yourcomputer.local`. On OSX you can see your computer's Bonjour name by going to System Preferences -> Sharing and
-looking under the Computer Name field.
+5. When asked to provide a challenge password while generating the SSL certificate press enter to skip setting a password.
 
-6. When asked to provide a challenge password while generating the SSL certificate press enter to skip setting a password.
+6. Copy the `config-example.js` file to `config.js` and change any options you deem necessary for your server.
 
-
-**Remember to configure the config.js file with your MySQL settings and server mode options.**
-
-The MySQL database tables are provided in **janusvr.sql**
+7. Run `mysql janusvr.sql`. This will install the MySQL database tables provided in **janusvr.sql**.
```

Comment:
This line is no longer relevant.

the plugins will create the tables if they don't exist.